packages/trpc/server/routers/viewer/slots/util.ts

```diff
@@ -423,6 +446,9 @@ export async function getAvailableSlots({ input, ctx }: GetScheduleOptions): Pro
           return usersArray;
         }, [] as (GetAvailabilityUser & { isFixed: boolean })[]);
 
+  if (attendeeUsers.length) {
+    usersWithCredentials.push(...attendeeUsers.map((user) => ({ ...user, isFixed: false })));
```

Comment:
```suggestion
    usersWithCredentials.push(...attendeeUsers.map((user) => ({ ...user, isFixed: true })));
```
Here we want isFixed to be true since want all attendees to strictly be available at the time slot

Reply:
Made it as true